src/FullBuild/CLI/CommandLine.fs

```diff
@@ -110,6 +110,11 @@ type private Token =
     | UpdateGuids
     | Migrate
 
+let (|FullBuildView|_|) (viewFile : string) =
+    if viewFile.EndsWith(IoHelpers.Extension.View |> IoHelpers.GetExtentionString |> sprintf ".%s") && System.IO.File.Exists(viewFile) then

```

Comment:
Hum? You mean move the logic of verifying if the file name ends by .fbv to the Views.Factory ?